cli/config/config.go

```diff
@@ -84,8 +84,14 @@ func LoadFromReader(configData io.Reader) (*configfile.ConfigFile, error) {
 	return &configFile, err
 }
 
-// Load reads the configuration files in the given directory, and sets up
-// the auth config information and returns values.
+// Load reads the configuration file ([ConfigFileName]) from the given directory.
```

Comment:
what is ([ConfigFileName]) here?

Reply:
They're "doc links"; if you use brackets and put a reference to a type, then GoDoc renders them as links.

You can test those using `pkgsite` (`go install golang.org/x/pkgsite/cmd/pkgsite@latest`), and then browse the docs (it's a bit fiddly on the CLI because of go modules, so I just ran it in a container);

<img width="1469" alt="Screenshot 2024-05-21 at 16 16 32" src="https://github.com/docker/cli/assets/1804568/9b8e250e-a0f0-4caf-8e1a-aff1779568a4">
<img width="1257" alt="Screenshot 2024-05-21 at 16 16 52" src="https://github.com/docker/cli/assets/1804568/ca7dc74b-3e6c-4471-a321-5811b4f43707">

Reply:
Ah, good to know! I didn't think that having just the type name would be enough for the linking to occur